### PR TITLE
Update metricbeat manifest

### PR DESCRIPTION
**Pull request by exekias**
This add new needed permissions and settings for versions >= 6.4

### CODE REVIEW

**Comment by exekias** [comment]
@ruflin you may want to have a look to this

**Review by ruflin — APPROVED** [review]
LGTM. Will test locally as soon as in.

File: manifests/metricbeat.yaml [hunk 1]
```diff
@@ -162,15 +169,28 @@ metadata:
...
+  resources:
+  - replicasets
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["apps"]
+  resources:
+  - statefulsets
+  - deployments
+  verbs: ["get", "list", "watch"]
+- apiGroups:
+  - ""
```
Comment (ruflin) [inline manifests/metricbeat.yaml:189]: Is this correct? Likes like nothing here for the array in yaml.

Comment (exekias) [reply manifests/metricbeat.yaml:189]: yes, "" indicates the core API group

Comment (exekias) [reply manifests/metricbeat.yaml:189]: but I agree, I wish this was just `"core"` :smile:

Comment (ruflin) [reply manifests/metricbeat.yaml:189]: Got it. Also got confused that on line 172 and here you use a different syntax

Comment (exekias) [reply manifests/metricbeat.yaml:189]: uhm, that's a good catch, these could be merged indeed, will need to test that

**Pull request merged**